lib/evss/gi_bill_status/gi_bill_status_response.rb

```diff
@@ -38,19 +45,27 @@ def initialize(status, response = nil, timeout = false, content_type = 'applicat
         super(status, attributes)
       end
 
+      def success?
+        contains_education_info?
+      end
+
+      def error_type
+        KNOWN_ERRORS.each do |_error_key, error_val|
+          return error_val if send("#{error_val}?")
+        end
+
+        'unknown'
+      end
+
+      private
+
+      ### ERROR types ###
```

Comment:
I'd remove the comments around the 'error?' methods I think it's self documenting. Could postfix `_error?` to all of them if you think their intention is not clear.